src/dials/command_line/rs_mapper.py

```diff
@@ -188,12 +188,13 @@ def process_imageset(self, imageset):
         beam = imageset.get_beam()
         s0 = beam.get_s0()
         pixel_size = panel.get_pixel_size()
-        xlim, ylim = imageset.get_raw_data(0)[0].all()
+        nfast, nslow = panel.get_image_size()
+
```

Comment:
To avoid any further confusion, this is the critical change. The rest is just cosmetic.